packit/local_project.py

```diff
@@ -12,6 +12,25 @@
 
 
 class LocalProject:
+    """
+    Class representing the cloned repository and the remote API
+
+    - git_repo: instance of git.Repo
+    - working_dir: working directory for the project
+    - ref: git ref (branch/tag/commit) if set, then checkouted
+    - git_project: instance of ogr.GitProject (remote API for project)
+    - git_service: instance of ogr.GitService (tokens for remote API)
+    - git_url: remote url (used for cloning)
+    - full_name: "$namespace/$repo"
+    - namespace: namespace of the remote project
+    - repo_name: name of the remote project
+    - path_or_url: working_dir if the directory exists
```

Comment:
how about we removed this parameter and performed the check in a dedicated function? then we could pass the path or url to LocalProject

Reply:
In this case, we need to check this in multiple places. (All CLI functions.)